fili-wikipedia-example/README.md

```diff
@@ -2,19 +2,44 @@ Fili Wikipedia Example Application
 ==================================
 
 This example is an entirely self contained example that provides a Fili application for the 
-[Druid Wikipedia example](http://druid.io/docs/0.9.1.1/tutorials/quickstart.html).
+[Druid Wikipedia example](http://druid.io/docs/latest/tutorials/quickstart.html).
 
 ## Setup and Launching
 
-1. Follow the [Druid instructions](http://druid.io/docs/0.9.1.1/tutorials/quickstart.html) to bring up a local Druid 
+1. Follow the [Druid instructions](http://druid.io/docs/latest/tutorials/quickstart.html) to bring up a local Druid 
    cluster on your Unix based machine.
+   
 2. Clone this repository to your computer.
+    ```bash
+    git clone git@github.com:yahoo/fili.git
+    ```
 3. Use Maven to install and launch the Fili Wikipedia example:
 
-```bash
-mvn install
-mvn -pl fili-wikipedia-example exec:java -Dexec.mainClass=com.yahoo.wiki.webservice.application.WikiMain
-```
+    ```bash
+    cd fili
+    mvn install
+    mvn -pl fili-wikipedia-example exec:java
+    ```
+    From another window, run a test query against the default druid data:
+
+    ```bash
+    curl "http://localhost:9998/v1/data/wikipedia/hour/?metrics=deleted&dateTime=2015-09-12/PT2H" -H "Content-Type: application/json" | python -m json.tool
+    ```
+    If everything is working you should see something like:
+    ```
+    {
+        "rows": [
+            {
+                "dateTime": "2015-09-12 00:00:00.000",
+                "deleted": 1761.0
+            },
+            {
+                "dateTime": "2015-09-12 01:00:00.000",
+                "deleted": 16208.0
+            }
+        ]
+    }        
+    ```
 
 ## Example Queries
```

Comment:
Maybe we can make the example queries include the `GET` method and a grey box around it with this PR?